apps/src/music/blockly/blocks/events.js

```diff
@@ -49,7 +50,7 @@ export const triggeredAt = {
 export const triggeredAtSimple = {
   definition: {
     type: BlockTypes.TRIGGERED_AT_SIMPLE,
-    message0: '%1 triggered',
+    message0: `%1 ${musicI18n.blockly_blockTriggered()}`,
```

Comment:
Should we incorporate the `%1` value into the loc'ed string, in case there is a translation where the `%1` value won't appear first?  Also it gives the translator more context.

Reply:
Similar comment for a few similar cases below.

Reply:
Yeah good question, wasn't sure about this. But I did end up doing this for a few strings that had `%1` in the middle of the string (ex. `play %1 at %2`), so it would be best to be consistent. I'll update this.